distribution/openrct2.d.ts

```diff
@@ -906,6 +912,8 @@ declare global {
         value: number;
     }
 
+    type RideCategory = "transport" | "gentle" | "rollercoaster" | "thrill" | "water" | "shop" | "";
```

Comment:
Thinking about `""`, I realize other enums like `RideClassification` don't include the empty string, even though it's possible in the code for the empty string to be passed under some circumstances. I might want to remove it. Thoughts?

Reply:
I'd remove it, seems like you wouldn't want to take any action on an empty type anyway.